less/paperhive/document.less

```diff
@@ -9,6 +9,14 @@
   }
 }
 
+.ph-document-journal {
+  max-width: 250px;
+  overflow: hidden;
+  display: inline-block;
+  text-overflow: ellipsis;
+  vertical-align: top;
+}
```

Comment:
This will work just fine, but some browser will interpret this different then others, using `line-height` in this case is a more common way. applying the half of the default `line-height: @line-height-base / 2;` does a great job. Even if you need the child elements to inherit the `line-height` property, unlike `vertical-align` doesnt inherit his property to the child elements.